src/rules/max_file_line_numbers.c

```diff
@@ -0,0 +1,12 @@
+#include <string.h>
+#include "rules/max_line_numbers.h"
+#include "arraylist.h"
+#include "display.h"
+
+unsigned int max_file_line_numbers(arraylist_t *file, unsigned int n){
+  if(file->size > n){
+    print_warning("fictive_file.c", n, arraylist_get(file, n), "Too much lines");
```

Comment:
"Too **many** lines"
Also, it would be better to show the total amount of lines compared to the maximum instead of the line itself in this case.